modules/ossm-control-plane-cli.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+[id="ossm-control-plane-deploy-cli_{context}"]
+= Deploying the control plane from the CLI
+
+Follow this procedure to deploy a basic `ServiceMeshControlPlane` from the command line.
+
+.Prerequisites
+
+* The {ProductName} Operator must be installed.
+* An account with the `cluster-admin` role.
```

Comment:
this shouldn't be required once the operator is installed

Reply:
@dgn I removed the requirement for an account with the cluster admin role.

The first step to log in "as a user with the `cluster-admin` role". Can I get rid of the as a user with the cluster admin role there too?

Reply:
Yep, that can be changed to just `log in to the console` -- any account that can create Projects will be sufficient